components/brave_vpn/common/wireguard/wireguard_utils.cc

```diff
@@ -48,6 +50,71 @@ constexpr char kWireguardConfigTemplate[] = R"(
   Endpoint = {vpn_server_hostname}:51821
 )";
 
+bool IsValidConfigDNS(const std::wstring& config_str) {
+  // Regex pattern to match our expected `DNS` field.
+  std::wregex endpoint_pattern(
+      LR"(^\s*DNS = 1.1.1.1$)",
```

Comment:
does the `.` need to be escaped here?

Reply:
oh wow - good call... yes, it should be escaped